### ISSUE TITLE
outdated and broken

### ISSUE THREAD

**Issue by clankill3r** [nytlabs/oauthp5#2]
The library is really outdated.
Using OAuth 1.0 for twitter etc.
Also the main page refers to bitbucket instead of github.

**Comment by zcanter** [comment]
Yeah, what a bummer. Such a great tool. Would be great to get it up and running again.
